examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1420,6 +1422,20 @@ class GLTFWriter {
 			map = decompress( map, options.maxTextureSize );
 
 		}
+		else if ( map.isRenderTargetTexture ) {
+			
+			if ( this.options.renderer ) {
+
+				const width = map.image.width;
+				const height = map.image.height;
+				const buffer = new Uint8Array(width * height * 4);
```

Comment:
Always assuming `RGBA8` is incorrect, see https://github.com/mrdoob/three.js/issues/28652#issuecomment-2169249094.